easybuild/easyconfigs/h/HTSeq/HTSeq-0.9.1-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,35 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = "PythonPackage"
+
+name = 'HTSeq'
+version = '0.9.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www-huber.embl.de/users/anders/HTSeq/'
+description = """A framework to process and analyze data from high-throughput sequencing (HTS) assays"""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+checksums = ['af5bba775e3fb45ed4cde64c691ebef36b0bf7a86efd35c884ad0734c27ad485']
+
+dependencies = [
+    ('numpy', '1.10.1', versionsuffix),
```

Comment:
Is this specific version of `numpy` required?

The `Python-2.7.12-foss-2016b` module already includes `numpy` v1.11.1...